### DISCUSSION TITLE
option to select multiple types of tests from "quote" tests, instead of only selecting all of them or just one of them

### DISCUSSION THREAD

**Discussion by 0xgurditt** [monkeytypegame/monkeytype#5862] [Ideas] [answered]
I would love to have the option to only select "medium" and "long" types of text for some tests, short tests for when I want to test my max speed. thicc/long tests for increasing endurance etc. Currently, I can only select one type of test, or all of them.
Follow up, would also love to have a "Randomize" option which randomly gives tests from all possible types of tests

**Comment by Miodec** [comment, marked as answer]
Shift click to select multiple
